secret/native/list.go

```diff
@@ -11,11 +11,11 @@ import (
 )
 
 // List captures a list of secrets.
-func (c *client) List(sType, org, name string, page, perPage int) ([]*library.Secret, error) {
+func (c *client) List(sType, org, name string, page, perPage int, teams []string) ([]*library.Secret, error) {
```

Comment:
Think we'll have to add a `nolint` comment here:

```suggestion
//
// nolint: lll // ignore long line length
func (c *client) List(sType, org, name string, page, perPage int, teams []string) ([]*library.Secret, error) {
```